### DISCUSSION TITLE
General Questions

### DISCUSSION THREAD

**Discussion by DeadpoolWasTaken** [andsfonseca/dota2-rpc-client#51] [Q&A]
Are you getting data using OpenDota API or are you fetching them from game itself?

**Comment by andsfonseca** [comment]
Hi @DeadpoolWasTaken, the data is taken from the Gamestate Integration API. You can see the documentation here: https://developer.valvesoftware.com/wiki/Counter-Strike:_Global_Offensive_Game_State_Integration

This software does not use any other type of resources other than the official Valve solution
